mbed_host_tests/__init__.py

```diff
@@ -57,6 +57,9 @@
 HOSTREGISTRY.register_host_test("wait_us_auto", WaitusTest())
 HOSTREGISTRY.register_host_test("dev_null_auto", DevNullTest())
 
+# Set the default baud rate
+DEFAULT_BAUD_RATE = 115200
```

Comment:
@bridadan 115200 was the previous value, but I agree that it should probably be changed to 9600. @mazimkhan?